README.md

```diff
@@ -14,55 +14,63 @@ Coder!
-| timescale                              | object | Contains configuration for the internal database. It is not recommended to run this service in production. See the `postgres` section for connecting to an external Postgres database.                                                                                                                                                       | `{"image":"","resources":{"limits":{"cpu":"250m","memory":"1Gi"},"requests":{"cpu":"250m","memory":"1Gi","storage":"10Gi"}}}` |
-| timescale.image                        | string | Injected during releases.                                                                                                                                                                                                                                                                                                                    | `""`                                                                                                                          |
-| timescale.resources                    | object | Kubernetes resource request and limits for the timescale pod. To unset a value, set it to "". To unset all values, you can provide a values.yaml file which sets resources to nil. See values.yaml for an example.                                                                                                                           | `{"limits":{"cpu":"250m","memory":"1Gi"},"requests":{"cpu":"250m","memory":"1Gi","storage":"10Gi"}}`                          |
-| timescale.resources.requests.storage   | string | Specifies the size of the volume claim for persisting the database.                                                                                                                                                                                                                                                                          | `"10Gi"`                                                                                                                      |
+| Key                                    | Type   | Description                                                                                                                                                                                                                                                                                                                                        | Default                                                                                                                       |
```

Comment:
is the README generated from somewhere? otherwise this seems like a lot of changes, and i haven't read through this

Reply:
yup! there's a script that does it.